script/deploy/comptroller/facet-cut-params-generator.ts

```diff
@@ -0,0 +1,109 @@
+import fs from "fs";
+import { ethers } from "hardhat";
+
+import { FacetCutAction, getSelectors } from "../../../tests/hardhat/Comptroller/Diamond/scripts/diamond";
```

Comment:
Would it make sense to move this `diamond.ts` file to this folder? This script (`facet-cut-params-generator.ts`) is not a test script. Importing a file from the `test` folder seems weird. Importing `diamond.ts` from a test file seems ok for me

Reply:
Moved, a1d991f909800df6505b9b451d4ca0e291c1ba9f